### DISCUSSION TITLE
Add archive and delete feature

### DISCUSSION THREAD

**Discussion by sreydray** [Radiokot/photoprism-android-client#136] [Ideas]
Please add the **archive and delete feature** like in the web interface. It will help delete multiple photos altogether hassle free.

**Comment by Radiokot** [comment]
Will be added in the next release 👌🏻

**Reply by sreydray** [reply]
cool

**Reply by Radiokot** [reply]
It's live in [1.27.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.27.0(41))
